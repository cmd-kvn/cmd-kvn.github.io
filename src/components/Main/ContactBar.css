nav img {
    height: 60px;
    width: 60px;
    padding: 20px;
}