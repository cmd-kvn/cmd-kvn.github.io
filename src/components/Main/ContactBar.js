import React from 'react';

import './ContactBar.css';

const ContactBar = () => {
    return (
        <nav>
            <img src='email-icon.png' alt='email icon' />
            <img src='github-icon.png' alt='github icon' />
            <img src='linkedIn-icon.png' alt='linkedIn icon' />
        </nav>
    );
};

export default ContactBar;
