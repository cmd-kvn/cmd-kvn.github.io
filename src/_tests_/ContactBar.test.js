import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ContactBar from '../components/Main/ContactBar';

describe('ContactBar', function() {
    it('runs the ContactBar', function() {
        const rendered = shallow(<ContactBar />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});
