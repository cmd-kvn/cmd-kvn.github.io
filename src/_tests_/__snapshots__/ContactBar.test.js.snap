exports[`ContactBar runs the ContactBar 1`] = `
<nav>
  <img
    alt="email icon"
    src="email-icon.png" />
  <img
    alt="github icon"
    src="github-icon.png" />
  <img
    alt="linkedIn icon"
    src="linkedIn-icon.png" />
</nav>
`;
